### PR TITLE
Add comment to explicit range_behavior attribute on Carrier class

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Added a small comment to make the attribute value more explicit
| Type?             | improvement
| Category? | CO
| BC breaks?        | no
| Deprecations?     | no

### CODE REVIEW

File: classes/Carrier.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ class CarrierCore extends ObjectModel
     /** @var bool True if extra shipping handling cost should be applied to this Carrier */
     public $shipping_handling = true;
 
-    /** @var bool Behavior taken for unknown range */
+    /** @var bool Behavior for out-of-range weights : true to disable carrier, false to apply the cost of the highest defined range */
```
Comment (kpodemski) [inline classes/Carrier.php]:
```suggestion
    /** @var bool Behavior for out-of-range weights: true to disable carrier, false to apply the cost of the highest defined range */
```

remember about no space before `:` :D

Comment (matks) [reply classes/Carrier.php]: 🆙 @thomasnares

Comment (thomasnares) [reply classes/Carrier.php]: Oups thanks, didn't got the notif

**Review by kpodemski — APPROVED** [review]

**Review by zuk3975 — APPROVED** [review]

**Comment by zuk3975** [comment]
thanks @thomasnares :+1:

**Pull request merged**